### PR TITLE
dep: introduce

### PR DESCRIPTION
**Pull request by kopiczko**
This also updates microstorage with EmptyList test.

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#  name = "github.com/x/y"
+#  version = "2.4.0"
+
+required = [
+  "k8s.io/api/core/v1",
+]
```
Comment (kopiczko) [inline Gopkg.toml]: Trick described in #news-dev

Comment (xh3b4sd) [reply Gopkg.toml]: Why do we need tricks now?

Comment (kopiczko) [reply Gopkg.toml]: Otherwise master version of the `k8s.io/api` is resolved, because there is no reference in the code.

Comment (xh3b4sd) [reply Gopkg.toml]: This seems to be a general problem, what about putting this hack into operatorkit? We have to do this now everywhere where we import operatorkit.

Comment (kopiczko) [reply Gopkg.toml]: I tried, but it doesn't work when it's imported: https://github.com/giantswarm/operatorkit/pull/145

**Pull request merged**